// Copyright 2019 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.
//
// Sync protocol datatype extension for the send tab to self entries.

syntax = "proto2";

import "components/sync/protocol/send_tab_to_self_specifics.proto";

option optimize_for = LITE_RUNTIME;

package send_tab_to_self;

// Local Send tab to self entry. This proto contains the fields stored locally
// for a send tab to self entry.
message SendTabToSelfLocal {
  // The Send tab to self specifics proto.
  optional sync_pb.SendTabToSelfSpecifics specifics = 1;

  // notification_dismissed has been moved into SendTabToSelfSpecifics, and is
  // deprecated inside this proto.
  reserved 2;
  reserved "notification_dismissed";
}
